etc/kayobe/pulp-repo-versions.yml

```diff
@@ -26,12 +26,17 @@ stackhpc_pulp_repo_rocky_9_2_baseos_version: 20231211T120328
 stackhpc_pulp_repo_rocky_9_2_crb_version: 20231207T013715
 stackhpc_pulp_repo_rocky_9_2_extras_version: 20231207T013715
 stackhpc_pulp_repo_rocky_9_2_highavailability_version: 20231208T011702
-stackhpc_pulp_repo_rocky_9_3_appstream_version: 20240416T010957
-stackhpc_pulp_repo_rocky_9_3_baseos_version: 20240413T014042
-stackhpc_pulp_repo_rocky_9_3_crb_version: 20240413T014042
-stackhpc_pulp_repo_rocky_9_3_extras_version: 20240413T014042
-stackhpc_pulp_repo_rocky_9_3_highavailability_version: 20240404T012937
+stackhpc_pulp_repo_rocky_9_3_appstream_version: 20240508T001430
+stackhpc_pulp_repo_rocky_9_3_baseos_version: 20240509T005935
+stackhpc_pulp_repo_rocky_9_3_crb_version: 20240429T003818
+stackhpc_pulp_repo_rocky_9_3_extras_version: 20240506T000343
+stackhpc_pulp_repo_rocky_9_3_highavailability_version: 20240510T001129
+stackhpc_pulp_repo_rocky_9_4_appstream_version: 20240704T001154
+stackhpc_pulp_repo_rocky_9_4_baseos_version: 20240707T011413
+stackhpc_pulp_repo_rocky_9_4_crb_version: 20240702T000233
+stackhpc_pulp_repo_rocky_9_4_extras_version: 20240707T235817
+stackhpc_pulp_repo_rocky_9_4_highavailability_version: 20240629T235004
 stackhpc_pulp_repo_rocky_9_sig_security_common_version: 20240708T235303
-stackhpc_pulp_repo_ubuntu_cloud_archive_version: 20240418T070026
-stackhpc_pulp_repo_ubuntu_jammy_security_version: 20240418T043733
-stackhpc_pulp_repo_ubuntu_jammy_version: 20240418T043733
+stackhpc_pulp_repo_ubuntu_cloud_archive_version: 20240709T043220
+stackhpc_pulp_repo_ubuntu_jammy_security_version: 20240709T015800
+stackhpc_pulp_repo_ubuntu_jammy_version: 20240709T015800
```

Comment:
I don't think we want to update jammy in this PR.

Reply:
Thanks, fixed.